lib/index.ts

```diff
@@ -0,0 +1,10 @@
+// NOTE: Aliases to classes and interfaces are used for external consumption.
+
+// SDK exports.
+import IonDid from './IonDid';
+import IonKey from './IonKey';
```

Comment:
Thinking to put external facing SDK classes into an "sdk" folder, or, moving all non-external facing classes into an `internal` folder. Not sure which one to go for, or if I am over-structuring it.

Reply:
To implementers, it shouldn't matter if something is internal or external I think, and to users of the SDK, they will just use what is exported and documented.

I think exporting the ones we want to expose and hiding the ones we don't is sufficient.

Reply:
This is more for us/developers. So you prefer to just have a flat list?